tests/models/xapi/fields/test_actor.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for the xAPI actor fields"""
+
+import pytest
+from pydantic import ValidationError
+
+from ralph.models.xapi.fields.actor import ActorField
+
+
+def test_models_xapi_fields_actor_actor_field_with_valid_content():
```

Comment:
Why not using hypothesis on this?

Reply:
I thought it would be simpler to use plain dictionaries in this case) Updated ActorField and ContextField tests to use hypothesis.